tests/components/automation/test_event.py

```diff
@@ -7,10 +7,11 @@
 from tests.common import get_test_home_assistant
 
 
+# pylint: disable=invalid-name
 class TestAutomationEvent(unittest.TestCase):
     """Test the event automation."""
 
-    def setUp(self):  # pylint: disable=invalid-name
+    def setUp(self):
```

Comment:
I also like the xunit style `setup_method(self, _):` which helps a bot with invalid-name